open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/metadataelements/ElementHeader.java

```diff
@@ -0,0 +1,287 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.dataengine.model.metadataelements;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.odpi.openmetadata.accessservices.dataengine.model.properties.MeaningProperties;
+import org.odpi.openmetadata.accessservices.dataengine.model.properties.SecurityTagsProperties;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+/**
+ * ElementHeader provides the common identifier and type information for all properties objects
+ * that link off of the asset and have a guid associated with them.  This typically means it is
```

Comment:
I am wondering whether the omas API documentation should refer to the omrs concepts like entity. The point of the omas is to expose an appropriate interface for a tool / persona; hiding away the more complicated internals.

Reply:
No longer applies, type doesn't exist